.changeset/smart-maps-wash.md

```diff
@@ -0,0 +1,10 @@
+---
+'@firebase/rules-unit-testing': patch
+'@firebase/auth-compat': patch
+'@firebase/firestore': patch
+'@firebase/functions': patch
+'@firebase/storage': patch
+'@firebase/auth': patch
```

Comment:
This should be invisible from the user's perspective but it is a big change under the hood, so I am leaning towards minor. If users notice any bugs or incompatibilities with the upgrade (we have tested this pretty thoroughly so it would be in an environment or use case we haven't thought of) it's feels like it makes more sense to point to this happening with a minor bump.

Reply:
Updated the changeset to `minor` updates for all of these SDKs.